3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -473,17 +473,22 @@ def initialize(debug, staging, enrich, shred, es, archive_raw, rdb_load, archive
           end
         end
 
+        latest_run_id = get_latest_run_id(s3, csbe[:good])
```

Comment:
I think it is hypothetically possible to have different run ids in `shredded` and `enriched`. Therefore `latest_run_id` must be evaluated for both steps separately (and only if we're in `recovery`)